src/templates/login/login.tsx

```diff
@@ -40,18 +56,58 @@ export const LoginTemplate: React.FC<LoginTemplateProps> = ({
         defaultMessage: "inloggen",
       });
 
+  const _titleLogin = titleLogin
+    ? titleLogin
+    : intl.formatMessage({
+        id: "mykn.templates.Login.titleLogin",
+        description: "templates.Login: The login title",
+        defaultMessage: "Welkom Terug",
+      });
+
+  const _titlePasswordForgotten = titlePasswordForgotten
+    ? titlePasswordForgotten
+    : intl.formatMessage({
+        id: "mykn.templates.Login.titlePasswordForgotten",
+        description: "templates.Login: The password forgotten title",
+        defaultMessage: "Wachtwoord vergeten?",
+      });
+
+  const _linkPasswordForgotten = linkPasswordForgotten
+    ? linkPasswordForgotten
+    : intl.formatMessage({
+        id: "mykn.templates.Login.linkPasswordForgotten",
+        description: "templates.Login: The password forgotten link",
+        defaultMessage: "Klik hier",
+      });
+
   return (
     <BaseTemplate
       columnProps={{ start: 5, span: 4 }}
-      pageProps={{ valign: "middle" }}
+      pageProps={{
+        valign: "middle",
+        backgroundImageUrl,
+      }}
       container={true}
       {...props}
     >
-      <Card>
+      <Card halign="center" shadow>
         <Body>
           {slotLogo || CustomLogo || <Logo />}
-          <Hr />
-          <Form labelSubmit={ucFirst(_labelLogin)} {...formProps} />
+          <br />
+          <br />
+          <H1>{_titleLogin}</H1>
+          <br />
+          <Form
+            labelSubmit={ucFirst(_labelLogin)}
+            {...formProps}
+            buttonProps={{
+              justify: true,
+            }}
+          />
+          <P muted>
```

Comment:
- The link should be customizable
- The P should only be shown when the link is provided.